xarray_array_testing/reduction.py

```diff
@@ -12,22 +13,15 @@ class ReductionTests(DuckArrayTestMixin):
     def expected_errors(op, **parameters):
         return nullcontext()
 
+    @pytest.mark.parametrize("op", ["mean", "sum", "prod", "std", "var"])
```

Comment:
One of "min" or "max" would be good for checking that dtype is preserved. I'm not sure "std" adds much over "var".

Reply:
These were really just chosen in order to cover some known bugs with cubed. Specifically that cubed doesn't yet implement `std` or `var`, and the array API standard only requires `mean` to work with floats, not integers.

Eventually we would want a test suite that covers everything, this is just meant to be a proof-of-principle for discussing overall design.